doc/articles/features/working-with-assets.md

```diff
@@ -85,6 +85,19 @@ The following languages have been verified to work on all platforms:
 \Assets\Images\logo.language-es.png
 ```
 
+### Dark theme support
+
+> Supported on Android only
+
+A theme qualifier can be specified for the image loader to use an asset based on the current app theme.
+
+#### Examples
+
+```
+/Assets/theme-light/ThemeTestImage.png
+/Assets/theme-dark/ThemeTestImage.png
```

Comment:
Is Assets/ThemeTestImage.theme-dark.png also valid? A quick glance on UWP Docs isn't clear, i'm not sure.  But if it works on UWP we should definitely support it since we do for all other qualifiers.

Reply:
https://docs.microsoft.com/en-us/windows/uwp/app-resources/images-tailored-for-scale-theme-contrast#qualify-an-image-resource-for-scale-theme-and-contrast

Reply:
The biggest change is in the ResourceConverter class that just moves the assets to its android counterpart. I believe that the ResourceConverter also looks for qualifiers in the filenames.